src/RPackage-Core/RPackage.class.st

```diff
@@ -693,9 +686,7 @@ RPackage >> importClass: aClass [
 
 	self basicImportClass: aClass.
 	self basicImportClass: aClass classSide.
-	self
-		addClassDefinition:  aClass
-		toClassTag: (self toTagName: aClass category)
+	self addClassDefinition: aClass toClassTag: aClass category
```

Comment:
Could we remove the category here?

Reply:
My main goal is to remove the categories but I need to simplify RPackage for that as a first step because it is way too complex